query/stdlib/testing/testing.go

```diff
@@ -46,12 +46,15 @@ var FluxEndToEndSkipList = map[string]map[string]string{
 		"window_null": "failed to read meta data: panic: interface conversion: interface {} is nil, not float64",
 
 		// these may just be missing calls to range() in the tests.  easy to fix in a new PR.
-		"group_nulls":         "unbounded test",
-		"integral":            "unbounded test",
-		"integral_columns":    "unbounded test",
-		"map":                 "unbounded test",
-		"join_missing_on_col": "unbounded test",
-		"rowfn_with_import":   "unbounded test",
+		"group_nulls":           "unbounded test",
+		"integral":              "unbounded test",
+		"integral_columns":      "unbounded test",
+		"map":                   "unbounded test",
+		"join_missing_on_col":   "unbounded test",
+		"join_use_previous":     "unbounded test (https://github.com/influxdata/flux/issues/2996)",
+		"rowfn_with_import":     "unbounded test",
+		"merge_filter_flag_on":  "unbounded test (https://github.com/influxdata/flux/issues/2995)",
+		"merge_filter_flag_off": "unbounded test (https://github.com/influxdata/flux/issues/2994)",
```

Comment:
Are these skipped tests?

Reply:
Yeah, we had three unbounded tests (though one being unbounded is an actual bug). The indent of the values makes it look bigger as a diff than it should be. Only three tests added.

Reply:
Do we have an issue for the one that's a bug? Just curious.

Reply:
Yeah, for all three skipped tests, I filed a new issue (per skipped test protocol). This is the one that's a bug: https://github.com/influxdata/flux/issues/2996

Reply:
thanks